### ISSUE TITLE
Can't get package from npm

### ISSUE THREAD

**Issue by replete** [yargalot/gulp-accessibility#1]
404: https://registry.npmjs.org/gulp-accessibility/-/gulp-accessibility-1.1.1.tgz

Any ideas?

**Comment by replete** [comment]
Just realised I can use `npm install yargalot/gulp-accessibility` to install from github. Look forward to trying out your work.

**Comment by replete** [comment]
Whilst the above shortcut installed gulp accessibility from gulp, I cannot get the package to actually work.

```
[20:04:58] Error: Cannot find module 'gulp-accessibility'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.defineProperty.get (/Users/phil/Dropbox/Projects/Cogent/dignity-funerals/Frontend/node_modules/gulp-load-plugins/index.js:48:18)
    at Gulp.<anonymous> (/Users/phil/Dropbox/Projects/Cogent/dignity-funerals/Frontend/gulpfile.js:182:12)
    at module.exports (/Users/phil/Dropbox/Projects/Cogent/dignity-funerals/Frontend/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/phil/Dropbox/Projects/Cogent/dignity-funerals/Frontend/node_modules/gulp/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/phil/Dropbox/Projects/Cogent/dignity-funerals/Frontend/node_modules/gulp/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/phil/Dropbox/Projects/Cogent/dignity-funerals/Frontend/node_modules/gulp/node_modules/orchestrator/index.js:134:8)
```

**Comment by TheDutchCoder** [comment]
Same issue here on Windows 7, regular install seems to work fine, but as soon as the plugin is called, it will fail with:

```
Error: Cannot find module 'gulp-accessibility'
```

**Comment by hgrimelid** [comment]
I get the "cannot find module" message, as well. Using version 1.1.2 with node 0.12.4.

**Comment by yargalot** [comment]
Wield, I'll have a look this weekend and sort it out. Sorry guys :(

**Comment by replete** [comment]
Hope you can look at it. Would be nice to at least try it out! Theres not many tools for gulp. I don't think I even got `gulp-a11y` working.

**Comment by labbydev** [comment]
+1

**Comment by TheDutchCoder** [comment]
@replete I have a11y working, but outside of gulp (it's not worth running it from gulp to be honest). let me know if you need help with it!

**Comment by yargalot** [comment]
Ok, one more shot, Gave it another whirl with an seperate repo and it seemed to work

Just note to not fail the build when testing you need to include 

```
access({
      force: true
})
```
